recipes/panic/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set name = "panic" %}
+{% set version = "9.4.2" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: panic-suite
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/panic-{{ version }}.tar.gz
+  sha256: f2772142ac60f69a8927a98317213804abb5becff565a852438288f97353827b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  # hint from cibuildwheel
+  script_env:
+    - MACOSX_DEPLOYMENT_TARGET=10.9
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ stdlib("c") }}
+    - python
+    - pip
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - fandango
+    - pytango
+
+test:
+  imports:
+    - panic    
+
+outputs:
+  - name: panic
+    build:
+      entry_points:
+        - PyAlarm = panic.ds.PyAlarm:main  
```

Comment:
You need the script to build the package here.

You also need all entry points here. You can't split the python package itself in different outputs.
For python multiple outputs, you always have one package with the full pyton code. Other outputs only have extra run requirements.